.gitignore

```diff
@@ -83,4 +83,5 @@ __pycache__/
 
 out/
 
-public/static/
\ No newline at end of file
+public/static/
+.netrc.gpg
```

Comment:
What's this about?

Reply:
I did not add the public/static/ line. I think that has already been there. .netrc.gpg is a way for me to store my github credentials in an encrypted file on disk.